apps/cookbook/src/app/component-status/component-status.component.ts

```diff
@@ -432,6 +421,33 @@ export class ComponentStatusComponent implements OnInit, OnDestroy {
       })
     );
   }
+
+  getPageOfIssues(url: string) {
+    const options = {
+      headers: new HttpHeaders({
+        Authorization: 'token ' + environment.oauth.githubToken1 + environment.oauth.githubToken2,
+      }),
+      observe: 'response' as const,
+    };
+    return this.http.get<Observable<any>>(url, options).pipe(
+      map((res) => ({
+        issues: res.body,
+        next: this.next(res),
+      }))
+    );
+  }
+
+  private next(response: HttpResponse<Observable<any>>): string {
+    let url: string;
+    const link = response.headers.get('Link');
+    if (link) {
+      const next = link.match(/<([^>]+)>;\s*rel="next"/);
+      if (next) {
+        [, url] = next;
```

Comment:
Very nice and concise syntax 👌🏻 
But consider adding a comment for the uninvited/future you/me...
```suggestion
      const next = link.match(/<([^>]+)>;\s*rel="next"/); // Omitting the global flag /g to ensure structure of match
      if (next) {
        [, url] = next; // If there's a match, the first item will be the complete match, second item the url capture group
```

Reply:
Thanks for helping decipher this!
I'll add it :)